script-library/jupyterlab-debian.sh

```diff
@@ -43,18 +44,15 @@ sudoUserIf()
   fi
 }
 
-# If we don't yet have Python, install it now.
-if ! python --version > /dev/null ; then
+# Make sure that Python is installed correctly
+if ! $PYTHON --version > /dev/null ; then
```

Comment:
nit: wrap in brackets in case the provided variable has a space in it, etc..

```suggestion
if ! "${PYTHON}" --version > /dev/null ; then
```

Reply:
and in other places